src/features/account/screens/login.screen.js

```diff
@@ -49,10 +35,10 @@ const OrSection = styled.View`
 `;
```

Comment:
this code and on line 68 seem not used. If you think this is not used, you probably can remove it. We don't want to see unused code floating in this codebase.